include/extractor/scripting_environment_lua.hpp

```diff
@@ -49,8 +52,8 @@ struct LuaScriptingContext final
 class Sol2ScriptingEnvironment final : public ScriptingEnvironment
 {
   public:
-    static const constexpr int SUPPORTED_MIN_API_VERSION = 0;
-    static const constexpr int SUPPORTED_MAX_API_VERSION = 1;
+    static const constexpr int SUPPORTED_MIN_API_VERSION = 1;
```

Comment:
there was never any code for handling version 0. m but we can allow it and make the api behave as with version 1 if you like.